packages/react-scripts/template/src/components/example/WagonWheelControl.js

```diff
@@ -23,7 +23,7 @@ export const WheelSpeedOptions = {
 }
 
 const colorCellCss = css`
-  height: var(--cell-width);
+  height: calc(var(--cell-width, 740px) / 2);
```

Comment:
is there a way to do this without a magic number, or is this temporary or a valid usage of a magic number? I have no context, just asking.

Reply:
Yeah, I'll assign them. Fixed.